core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackToStackV4ResponseConverter.java

```diff
@@ -54,6 +40,18 @@
 import com.sequenceiq.cloudbreak.service.stack.StackService;
 import com.sequenceiq.common.api.telemetry.model.Telemetry;
 import com.sequenceiq.common.api.telemetry.response.TelemetryResponse;
+import org.slf4j.Logger;
```

Comment:
please change your IDE's settings to follow the same import order as others. Hint: config/idea/idea_settings.jar

Reply:
done